src/components/Shared/form/Select.tsx

```
@@ -58,9 +81,12 @@ function reducer(state: State, action: Action): State {
     case 'SET_NEW_OPTIONS':
       return { ...state, newOptions: action.payload }
     case 'ADD_OPTION':
+      //@ts-expect-error Options[number] does not fully satisfy Options. This means that the newly added Option will not show up in the Options-type.
+      const options = [...state.newOptions, action.payload] as Options
```

Biome (2.4.7)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)